frontend/src/metabase/lib/api.js

```diff
@@ -29,7 +28,7 @@ const DEFAULT_OPTIONS = {
 };
 
 export class Api extends EventEmitter {
-  basename = "";
+  basename = window.location.origin;
```

Comment:
Need to double check whether this breaks anything e.g. embedding

Reply:
https://developer.mozilla.org/en-US/docs/Web/API/Fetch_API/Cross-global_fetch_usage